controllers/storagecluster/external_resources_test.go

```diff
@@ -23,7 +24,29 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-var ExternalResources = []ExternalResource{
+/*
```

Comment:
What is this comment? Can you add a comment about the comment?

Reply:
The comment was regarding a sample expected JSON input from UI. Removed it.